src/models/Like.model.ts

```diff
@@ -0,0 +1,21 @@
+import { ObjectId } from "mongodb";
+import { model, Schema, Document } from "mongoose";
+
+export interface Like extends Document {
+  userId: ObjectId;
+  tweetId: ObjectId;
```

Comment:
```suggestion
  user: ObjectId;
  tweet: ObjectId;
```
@alexandrtovmach ordered delete "Id" at all models